### ISSUE TITLE
Pointing while left-right is turned on and rotation center is off causes weird behavior...

### ISSUE THREAD

**Issue by Toddap** [jmoenig/Snap--Build-Your-Own-Blocks#1454]
Repro bug:
http://snap.berkeley.edu/snapsource/snap.html#present:Username=juanitaintro&ProjectName=Mario%20-%20Bug 

Just view the code and click on the two "point in direction" blocks....   The Mario sprite jumps up and down on the y axis even though the y position (and costume) don't change.

This happens (apparently) b/c the rotation center is at the bottom of the sprite and snap apparently is doing something odd/wrong when it does the flip from left to right action.   BYOB does not have this behavior FWIW.

**Comment by jmoenig** [comment]
Yes, that's because the rotation center is at the bottom of the costumes in your project, and that's a feature. When you import the costumes the rotation center is automatically placed in the middle of a costume, that should work fine for your project.

You can also reset the rotation center's y-coordinate programmatically:

![reset_rotation_y](https://cloud.githubusercontent.com/assets/2596285/19210780/0607f8d4-8d2d-11e6-93b1-747d9708d08f.png)
